{"name": "multiply_3000_2000", "status": "failed", "statusDetails": {"message": "AssertionError: assert 60000000 == 6000000\n  +60000000\n  -6000000", "trace": "self = <testing.test_Calculator.TestCalc object at 0x000001564F469948>, get_instance = <pythoncode.Calculator.Calculator object at 0x000001564F4692C8>, get_multiply_int_datas_with_fixture = [3000, 2000, 60000000]\n\n    @allure.title(\"multiply_{get_multiply_int_datas_with_fixture[0]}_{get_multiply_int_datas_with_fixture[1]}\")\n    @allure.story(\"multiply Integer Function\")\n    def test_multiply_int(self, get_instance, get_multiply_int_datas_with_fixture):\n        multiply_int = get_multiply_int_datas_with_fixture\n>       assert multiply_int[2] == get_instance.multiply(multiply_int[0], multiply_int[1])\nE       assert 60000000 == 6000000\nE         +60000000\nE         -6000000\n\ntest_Calculator.py:58: AssertionError"}, "attachments": [{"name": "log", "source": "43b21a24-6fae-45fc-8f44-45279be2baad-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "get_multiply_int_datas_with_fixture", "value": "[3000, 2000, 60000000]"}], "start": 1612420386908, "stop": 1612420386909, "uuid": "ed4c1eb4-bc92-4b14-9842-eeb989a8fa69", "historyId": "dd34b1cccd21045bf2ad03024137e43a", "testCaseId": "74ae4a99e2b1463598db95f6c0fec520", "fullName": "test_Calculator.TestCalc#test_multiply_int", "labels": [{"name": "feature", "value": "计算器"}, {"name": "story", "value": "multiply Integer Function"}, {"name": "suite", "value": "test_Calculator"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "DESKTOP-ALON4FT"}, {"name": "thread", "value": "10176-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_Calculator"}]}